content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -11,13 +11,7 @@ weight: 30
 {{% capture overview %}}
 This page explains how to add versioning information to
 [CustomResourceDefinitions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#customresourcedefinition-v1beta1-apiextensions), to indicate the stability
-level of your CustomResourceDefinitions. It also describes how to upgrade an
-object from one version to another.
-
-{{< note >}}
-**Note**: All specified versions must use the same schema. The is no schema
-conversion between versions.
-{{< /note >}}
+level of your CustomResourceDefinitions or advance your API to a new version with conversion between API Onbjects. It also describes how to upgrade an object from one version to another.
```

Comment:
no really conversion "between API objects". The object is always the same, the repesentation differs.